iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/gvl/Vendor.java

```diff
@@ -39,6 +40,14 @@
     private String policyUrl;
     private Instant deletedDate;
     private com.iabtcf.extras.gvl.Overflow overflow;
+    private Integer cookieMaxAgeSeconds;
+    @JsonProperty("usesCookies")
```

Comment:
why do some fields have JsonProperty while others don't?

Reply:
if we are ok having the method name as getUsesCookies() instead of usesCookies, then we can get rid of the JsonProperty annotation. As the method returns a boolean, a getter doesn't look good, but I can change it (in order to get rid of the annotation). The annotation is helping the jackson parser to identify the field name in the JSON.